src/common/utils/functional/functional.ts

```diff
@@ -31,11 +31,11 @@ export type Predicate<T> = Unary<T, boolean>;
 export function noop(...args: any[]): any {
 }
 
-export const identity = <T>(x: T): T => x;
+export const identity = <T,>(x: T): T => x;
```

Comment:
Why? Some issues with jsx vs ts? We can rewrite these functions as non-arrow if needed.

Reply:
As mentioned [here](https://esbuild.github.io/content-types/#typescript) it's a case when using generics and tsx loader. 
Generic type parameters | <T>(a: T): T => a | Must write <T,>(... with the tsx loader
-- | -- | --